lib/jsonld-signatures.js

```diff
@@ -14,6 +14,33 @@ Object.assign(api, constants);
 // TODO: support `ProofChain`
 const ProofSet = require('./ProofSet');
 
+/**
```

Comment:
Typically the jsdoc has a one liner for a function description.

Reply:
Added.